static/js/directives/content-row.js

```diff
@@ -17,10 +17,7 @@ angular.module('inboxDirectives').directive('mmContentRow', function() {
       read: '=',
 
       // string: (optional) the id of the resource icon to render
-      resourceIcon: '=',
-
-      // string: (optional) the class of the font awesome icon to render
-      fontIcon: '=',
+      icon: '=',
```

Comment:
Seems weird to have an object where every value is the same!

Reply:
Yeah... angular directives are a bizarre DSL. The `=` is shorthand for "icon: 'icon'" meaning don't rename the variable in the inner context.